jwql/website/manage.py

```diff
@@ -36,10 +36,25 @@
 import os
 import sys
 
+from jwql.utils.utils import get_config()
+
 if __name__ == "__main__":
 
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "jwql_proj.settings")
 
+    directory_mapping = {
+        'filesystem': 'filesystem',
+        'outputs': 'outputs',
+        'preview_image_filesystem': 'preview_images',
+        'thumbnail_filesystem': 'thumbnails'
+        }
+
+    for directory in ['filesystem', 'outputs', 'preview_image_filesystem', 'thumbnail_filesystem']:
+        symlink_location = os.path.join(get_config()['jwql_dir'], directory_mapping[directory])
```

Comment:
If I am understanding this correctly, `symlink_location` should be pointing to the location in which the symlinks will be written _within the local clone of the repo_.  If that is the case, then I think this line needs to be changed to be relative to the local clone's `static` path, instead of the `jwql_dir`, as `jwql_dir` points to the central storage location.

Reply:
Yes, that's definitely correct, and I should have picked that up. The existing code areas that use the symlinks do things via the config file, and I expect it's exactly the same when it comes to the active server, but we should do it differently. Is there a standard way to refer to the local directory, or should I be building a path based on the location of __file__?

Reply:
Right, the best way I can think to do it at the moment is building a path based on the file location, something like this:

```python
symlink_location = os.path.join(os.path.dirname(__file__), 'apps/jwql/static/', directory_mapping[directory)
```

Reply:
Fixed I think.